app/components/signup-steps/step-zero.hbs

```diff
@@ -0,0 +1,26 @@
+<div class='logos-container' data-test-logos-container>
+  <div class='logo logo--github' data-test-github-logo>

```

Comment:
`--`  is used as modifier in BEM, but I am not seeing it usable here.
Let me know why you use `logo--github` instead of `logos-container__github`

Reply:
Done with changes can you please check latest commit